File: test/fixtures/output/php/guzzle6/multipart-form-data.php
<?php

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har', [
    'multipart' => array(
        array(
            'foo' => 'bar',
        ),
    ),
]);

echo $response->getBody();

File: src/targets/rust/reqwest/fixtures/multipart-form-data.rs
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "http://mockbin.com/har";

    let form = reqwest::multipart::Form::new()
        .text("foo", "bar");
    let mut headers = reqwest::header::HeaderMap::new();

    let client = reqwest::Client::new();
    let response = client.post(url)
        .multipart(form)
        .headers(headers)
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    dbg!(results);
}
